### PR TITLE
[DD-315]: Add script for doc site pages stats

### PR DESCRIPTION
**Pull request by rsylvian**
# Description
- Add a `pages_stats_csv` directory in `scripts/` that contains all the code to run the page's analysis

# Reasons
This is very similar to the work that was done in https://github.com/circleci/circleci-docs/pull/5959. We want to collect data about our pages (such as last updated date, snippets count ...) to better understand the current state of the Doc. This script is using a mix of glob and git to extract this data and export it as a CSV file that can later be imported in Google sheet.

### CODE REVIEW

File: scripts/pages_stats_csv/index.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  };
+};
+
+const explore = async () => {
+  let promises = [];
+
+  const files = await glob(
+    `${repoPath}/jekyll/@(${directories.join('|')})/*.@(md|adoc)`,
+  );
+  log(`Found ${files.length} files`);
```
Comment (chantelle11) [inline scripts/pages_stats_csv/index.js:45]: Should we clean up logs?

**Review by chantelle11 — APPROVED** [review]
Nice work! 🤯🧚👌

**Review by chantelle11 — COMMENTED** [review]
I seem to be having issues running the script
```
=> jekyll/_cci2_ja/collect-test-data.md has 53 snippets
(node:3475) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'lineNo' of undefined
at /scripts/snippets_csv/index.js:88:66
```

**Review by tshen815 — APPROVED** [review]
tested and created a .csv file locally. Looks great!

**Review by teesloane — APPROVED** [review]

**Review by nramabad — APPROVED** [review]

**Review by chantelle11 — APPROVED** [review]
LGTM! ✌️🌺🧚

**Review by devinmarieb — APPROVED** [review]

**Pull request merged**